documents/Libraries/stdlib/stdlib_sx-glsl_impl.mtlx

```diff
@@ -679,22 +679,22 @@
   <!-- ======================================================================== -->
 
   <!-- <convert> -->
-  <implementation name="IM_convert_float_color2_sx_glsl" nodedef="ND_convert_float_color2" file="stdlib/sx-glsl/mx_convert_float_color2.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_float_color3_sx_glsl" nodedef="ND_convert_float_color3" file="stdlib/sx-glsl/mx_convert_float_color.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_float_color4_sx_glsl" nodedef="ND_convert_float_color4" file="stdlib/sx-glsl/mx_convert_float_color4.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_float_vector2_sx_glsl" nodedef="ND_convert_float_vector2" file="stdlib/sx-glsl/mx_convert_float_vector2.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_float_vector3_sx_glsl" nodedef="ND_convert_float_vector3" file="stdlib/sx-glsl/mx_convert_float_vector.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_float_vector4_sx_glsl" nodedef="ND_convert_float_vector4" file="stdlib/sx-glsl/mx_convert_float_vector4.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_vector2_color2_sx_glsl" nodedef="ND_convert_vector2_color2" file="stdlib/sx-glsl/mx_convert_vector2_color2.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_vector3_color3_sx_glsl" nodedef="ND_convert_vector3_color3" file="stdlib/sx-glsl/mx_convert_vector_color.inline"  language="sx-glsl"/>
-  <implementation name="IM_convert_vector4_color4_sx_glsl" nodedef="ND_convert_vector4_color4" file="stdlib/sx-glsl/mx_convert_vector4_color4.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_color2_vector2_sx_glsl" nodedef="ND_convert_color2_vector2" file="stdlib/sx-glsl/mx_convert_color2_vector2.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_color3_vector3_sx_glsl" nodedef="ND_convert_color3_vector3" file="stdlib/sx-glsl/mx_convert_color_vector.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_color4_vector4_sx_glsl" nodedef="ND_convert_color4_vector4" file="stdlib/sx-glsl/mx_convert_color4_vector4.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_color3_color4_sx_glsl" nodedef="ND_convert_color3_color4" file="stdlib/sx-glsl/mx_convert_color_color4.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_color4_color3_sx_glsl" nodedef="ND_convert_color4_color3" file="stdlib/sx-glsl/mx_convert_color4_color.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_boolean_float_sx_glsl" nodedef="ND_convert_boolean_float" file="stdlib/sx-glsl/mx_convert_boolean_float.inline" language="sx-glsl"/>
-  <implementation name="IM_convert_integer_float_sx_glsl" nodedef="ND_convert_integer_float" file="stdlib/sx-glsl/mx_convert_integer_float.inline" language="sx-glsl"/>
```

Comment:
Removing all the source code (inline) implementations for convert node. Since it's handled by a C++ class now instead.